db/db_impl/db_impl.cc

```diff
@@ -1304,6 +1312,12 @@ Status DBImpl::SetOptions(
   }
   sv_context.Clean();
 
+  if (s.ok() && (options_map.count("preserve_internal_time_seconds") > 0 ||
```

Comment:
Just for my own understanding. Even though db mutex is already released here. A race condition is avoided with the protection from `options_mutex_` since all other callers of `RegisterRecordSeqnoTimeWorker` also need to lock `options_mutex_` first and hold it atomically until all operations finish, right?

Reply:
That's right! :)